share/spice/crackhash/crackhash.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crackhash = function(api_result){
+
+	if(!api_result || !api_result.hits || api_result.hits.length === 0) {
+	    return Spice.failed('Crackhash');
+	}
+        Spice.add({
+            id: "crackhash",
+
+            name: "crackhash",
+            data: api_result.hits,
+            meta: {
+                sourceName: "crackhash",
+                sourceUrl: sourceUrl,
+	        itemType: (api_result.hits.length === 1) ? 'crack hash' : 'crack',
+   	        searchTerm: decodeURIComponent(query)
```

Comment:
Looks like the indentation is a little off here. Please ensure the file is consistently indented with 4-space soft tabs